docs/_data/en/landing_page_text.yml

```diff
@@ -3,7 +3,4 @@
hing, handling file locations, and building cross software functional tools.</li></ul><h3 id="want-to-automate-processes-around-the-shotgun-web-app-">Want to automate processes around the Shotgun web app?</h3><p>Shotgun offers a few technologies for adding triggers or custom menu items to your Shotgun site:</p><ul><li><strong>Action Menu Items</strong>, or AMIs allow you to add items to the context menus on entities, from which Shotgun users can run custom logic. (should we do examples?)</li><li><strong>Webhooks</strong> allow you to create custom reactions to events that take place on your Shotgun site. When a chosen event takes place, an HTTPS request is issued to an endpoint you define, with a payload of information about the event. You can then create custom code that uses this payload to react however you want. </li><li><strong>The Event Daemon framework</strong> is another solution for creating reactions to events on your site. Choose the Event Daemon framework if you prefer to limit incoming internet connections to your studio. </li></ul><h3 id="want-to-extend-shotgun-into-your-artist-pipeline-">Want to extend Shotgun into your artist pipeline?</h3><p>Shotgun Toolkit is  a platform for building plugin integrations within artists’ tools that bring the Shotgun experience into production workflows and give you the foundation for a publish pipeline.</p><p>There are many ways to customize your Toolkit setup:</p><ul><li>The Toolkit Pipeline Configuration is a set of YAML files that give you control of many options within Toolkit’s plugin apps.</li><li>Hooks allow you to inject custom Python logic at specific points in Toolkit’s workflows, without having to fork the main codebase.</li><li>The Toolkit Core API provides methods with which pipeline TDs can quickly build configurable tools that can run across common content creation software.</li><li>In addition, various app and framework APIs allow you to write code that handles specific Toolkit functionality, like publishing artist work, dealing with UI widgets, and various utilities.</li></ul><h3 id="want-to-customize-your-studio-s-media-review-experience-">Want to customize your studio’s media review experience?</h3><p>RV was built from the ground up to be an extensible platform. It provides a flexible system for adding automation, with an interface built primarily through its API provided in Mu, Python, and Javascript.  Some examples of custom behavior provided through plugins are:</p><ul><li>Automating Color Pipelines and View Management</li><li>Providing artist-facing tools within RV that utilize existing studio libraries</li></ul><h3 id="want-to-learn-more-about-real-world-ways-that-others-are-using-shotgun-">Want to learn more about real-world ways that others are using Shotgun?</h3><p>Our <a href="https://community.shotgunsoftware.com/">Community Forums</a> are a home for vibrant conversations among Shotgun staff and users, administrators, and developers working with Shotgun every day. </p>
```

Comment:
> (should we do examples?)

Is that for us reviewing this, or is it in there by mistake?

In general, I think it perhaps mentions Toolkit too much instead of Pipeline Integrations, but I'm just trying to think of what @robblau would say.

Also minor thing, but I'm still not keen on the blue circle Shotgun logo, it doesn't seem to fit with the rest of our branding, does it? Though I appreciate this isn't a change from what was there.

Overall I think this page looks really good!

Reply:
good catch - bad copy/paste job!
as for the logo, i think it's fine for now – perhaps a bigger design conversation is in order, but it does go with the blue that weaves through the site.